docker-compose.yml

```
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+
+  dev:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/app
+      - checkout_node_modules-vol:/app/node_modules
+    ports:
+      - 80:5000
+    entrypoint:
+      - /bin/sh
+      - ./startup.sh
+    env_file:
+      - ./env-docker
+    depends_on:
+      - redis
+      - rabbitmq
+    networks:
+      - net
+
+  redis:
+    image: redis:alpine
+    networks:
+      - net
+
+  rabbitmq:
+    image: rabbitmq:management
+    ports:
+      - target: 5672
+        published: 5672
+        protocol: tcp
+        mode: host
+      - target: 15672
+        published: 15672
```

suggestion(syntax):  I think you could use the _short-form_ syntax here, which would be a little easier to read & be consistent above with the port declaration for the `dev`/flask container.

https://docs.docker.com/compose/compose-file/compose-file-v3/#short-syntax-1

```suggestion
      - 5672:5672
      - 15672:15672
```

---

good call, updated accordingly